README.md

```diff
@@ -33,30 +33,74 @@ Prerequisites
     $ cd libtock-c
     ```
 
-1. The main requirement to build the C applications in this repository
-   is a cross compiler for embedded targets. Currently, the only
-   supportetd target is ARM Cortex M, so you will need an
-   `arm-none-eabi` toolchain.
+1. The main requirement to build the C applications in this repository is having
+   cross compilers for embedded targets. You will need an `arm-none-eabi`
+   toolchain for Cortex-M targets, and `riscv64-unknown-elf` toolchain for
+   RISC-V targets.
 
-   MacOS:
+   **MacOS**:
    ```
    $ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain --with-multilib
    ```
 
-   Ubuntu (18.04LTS or later):
+   You can also download a pre-built RISC-V toolchain
+   [here](http://cs.virginia.edu/~bjc8c/archive/gcc-riscv64-unknown-elf-8.3.0-mac.zip).
```

Comment:
This unfortunately doesn't work unless you have all of the dependent libraries also installed. Running this on my machine resulted in:

```
$ make
  CC        ../../libtock/crt0.c
  AR        ../../libtock/build/cortex-m0/libtock.a
  CC        ../../libtock/crt0.c
  AR        ../../libtock/build/cortex-m3/libtock.a
  CC        ../../libtock/crt0.c
  AR        ../../libtock/build/cortex-m4/libtock.a
  CC        ../../libtock/internal/alarm_internal.c
dyld: Library not loaded: /usr/local/opt/isl/lib/libisl.21.dylib
  Referenced from: /private/tmp/g/gcc-riscv64-unknown-elf-8.3.0/bin/../libexec/gcc/riscv64-unknown-elf/8.3.0/cc1
  Reason: image not found
riscv64-unknown-elf-gcc: internal compiler error: Abort trap: 6 signal terminated program cc1
Please submit a full bug report,
with preprocessed source if appropriate.
See <https://gcc.gnu.org/bugs/> for instructions.
make: *** [../../libtock/build/rv32imac/alarm_internal.o] Error 4
```

Reply:
```
$ ls /usr/local/opt/isl/lib/
libisl.22.dylib libisl.a        libisl.dylib    pkgconfig
```

Or, apparently I've just updated something else more recently than you?

In either case, to ship a pre-built compiler image, we'd need to figure out how to statically link everything in the compiler build (if we can?) to be reliable across machines.

Reply:
Fair enough, probably better to just remove this link then.